src/events/cd-event-list.vue

```diff
@@ -91,6 +93,15 @@
         return totalEventCapacity === totalTicketsBooked;
       },
     },
+    watch: {
+      currentUser(newUser) {
+        if (newUser) {
+          DojosService.isUserMemberOfDojo(newUser.id, this.dojo.id).then((response) => {
+            this.isMember = !!response.body;
```

Comment:
Would be a bit cleaner to assign body to `this.usersDojos` or something like that, then have a computed property for `isMember`. Allows us to use the data in other ways in the future, like showing admin controls to admins.